caravel/controllers/moderation.py

```diff
@@ -50,6 +68,15 @@ def moderation_view():
         key.delete()
         return ""
 
+    if request.form.get("automod"):
+        if config.enabled:
```

Comment:
Moderation.enabled is the property, `Moderation.get().enabled` is the value of that property on the singleton instance. (`Moderation.get()` is a database call to retrieve the configuration, so we don't want to do it multiple times.)